### PR TITLE
Rebrand /iot/smart-displays

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- Apply rebranding on /internet-of-things/smart-displays
- No changes to content, only design

## QA

- Go to https://ubuntu-com-14538.demos.haus/internet-of-things/smart-displays
- Check that design matches [Figma](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=2001-15688&node-type=frame&m=dev)
- No changes to design

## Issue / Card

Fixes [WD-16639](https://warthogs.atlassian.net/browse/WD-16639)

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-16639]: https://warthogs.atlassian.net/browse/WD-16639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mattea-turic** [comment]
ty @britneywwc ! :) a couple assets changes, sorry!

For this section, for smaller screens:
Screenshot 2024-12-05 at 10 33 13
- [ ] Could you maintain the images pls?

For this section:
Screenshot 2024-12-05 at 10 24 59
- [ ] The logos look a lil small to me so I've uploaded new ones in the [assets server](https://assets.ubuntu.com/manager?tag=iot-smart-displays-assets) (they've got the suffix "-updated". The container for these is also a lot smaller, so it should look good on smaller screens to now

For this section:
Screenshot 2024-12-05 at 10 29 34
- [ ] The logos look too big to me, so I've added new assets, again with the suffix "-updated"
- [ ] Something strange is going on with this section on smaller screens btw:
Screenshot 2024-12-05 at 10 35 04

For this section, for smaller screens:
Screenshot 2024-12-05 at 10 36 20
- [ ] Could you maintain the images pls

**Comment by eliman11** [comment]
Thanks Britney looks great! Some comments:
- [ ] Fix typo under 'Get started now with Ubuntu Core and Frame' to "Your graphical application will instantly run _in_ full screen with all input modalities seamlessly on all hardware."
- [ ] Link 'More information about Ubuntu Frame graphics support ›' to https://discourse.ubuntu.com/t/where-does-ubuntu-frame-work/23193
- [ ] 'Get in touch ›' under 'Learn how we can get your device to market for you' to link to contact modal instead of page
- [ ] Change h5s into h3 without changing styling
Screenshot 2024-12-05 at 10 42 15

**Comment by eliman11** [comment]
Thanks for the changes @britneywwc, looks great! Adding +1

**Comment by mattea-turic** [comment]
ty @britneywwc and sorry for late pr review ;-;

I just had a couple questions:

- [ ] Would you be able to use the [wrap variant](https://vanillaframework.io/docs/examples/patterns/equal-height-row/4-items-per-column-wrap-responsive?theme=light) here for the equal heights row so that the aspect ratio self-adjusts for smaller screens, and so tablets use a 2x2 layout? – and same thing for the "ready for all boards..." section
Screenshot 2024-12-10 at 00 15 29

And then for this section:
Screenshot 2024-12-10 at 00 18 59
Screenshot 2024-12-10 at 00 19 50

- [ ] Could you reduce the size of the images, particularly for smaller screens? With the previous asset I provided, the logos were smaller but there was a lot of whitespace. Now, there is less whitespace, but the logos are v big :') is there a format of the logos I could send over that would work better for both?

All else is good tho!

**Comment by britneywwc** [comment]
@mattea-turic Thanks for the review. I'm not sure about the image dimensions for Ubuntu Core and Frame, maybe something similar to the images that are being used in the [Hero Signpost](https://vanillaframework.io/docs/examples/patterns/hero/hero-signpost)? Either that or I could hide the logos on smaller screens.

**Comment by mattea-turic** [comment]
> @mattea-turic Thanks for the review. I'm not sure about the image dimensions for Ubuntu Core and Frame, maybe something similar to the images that are being used in the [Hero Signpost](https://vanillaframework.io/docs/examples/patterns/hero/hero-signpost)? Either that or I could hide the logos on smaller screens.

Got it, I've added a resized version for [Ubuntu Core](https://assets.ubuntu.com/v1/00216ccc-ubuntu-core-resized.png) and for [Frame](https://assets.ubuntu.com/v1/45f813f2-ubuntu-frame-resized.png). Hopefully this'll work better now :)

File: templates/internet-of-things/smart-displays.html [hunk 1]
```diff
@@ -7,377 +7,541 @@
...
+                                height="1279",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class":"p-image-container__image"}) | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h5 class="p-heading--5">Secure</h5>
```
Comment (muhammad-ali-pk) [inline templates/internet-of-things/smart-displays.html:79]:
```suggestion
              <h3 class="p-heading--5">Secure</h5>
```

File: templates/internet-of-things/smart-displays.html [hunk 2]
```diff
@@ -7,377 +7,541 @@
...
+                                height="1279",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h5 class="p-heading--5">Reliable</h5>
```
Comment (muhammad-ali-pk) [inline templates/internet-of-things/smart-displays.html:102]:
```suggestion
              <h3 class="p-heading--5">Reliable</h5>
```

File: templates/internet-of-things/smart-displays.html [hunk 3]
```diff
@@ -7,377 +7,541 @@
...
+                                height="1279",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h5 class="p-heading--5">Simple</h5>
```
Comment (muhammad-ali-pk) [inline templates/internet-of-things/smart-displays.html:126]:
```suggestion
              <h3 class="p-heading--5">Simple</h5>
```

File: templates/internet-of-things/smart-displays.html [hunk 4]
```diff
@@ -7,377 +7,541 @@
...
+                </p>
+                <p>
+                  Ubuntu Core brings all the security features and device management solutions you need to deploy, provision, and maintain your device.
+                </p>
+              </div>
+              <div class="p-cta-block">
+                <a href="/core">Learn more about Ubuntu Core&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
```
Comment (muhammad-ali-pk) [inline templates/internet-of-things/smart-displays.html:242]: Wondering if it was possible to use tiered list pattern for this section

File: templates/internet-of-things/smart-displays.html [hunk 5]
```diff
@@ -7,377 +7,541 @@
...
-        <hr class="p-rule--muted u-hide--small" />
-        <h3 class="p-card__title">Broadsign</h3>
-        <p class="p-card__content">
-          An end-to-end software solution built on Ubuntu Core for media owners and buyers to plan, manage and deliver smart displays of any kind..
-        </p>
-        <a href="https://broadsign.com/">Learn more about Broadsign&nbsp;&rsaquo;</a>
+    <div class="row">
+      <div class="col-start-large-7 col-6">
+        <div class="row--50-50">
+          <div class="p-equal-height-row p-equal-height-row-50-50">
```
Comment (muhammad-ali-pk) [inline templates/internet-of-things/smart-displays.html:346]: `<div class="row--50-50">` seems unnecessary as the div below is also specifying row-50-50 i.e., `<div class="p-equal-height-row p-equal-height-row-50-50">`

File: templates/internet-of-things/smart-displays.html [hunk 6]
```diff
@@ -7,377 +7,541 @@
...
+          <div class="row--25-75">
+            <hr class="p-rule--muted" />
+            <div class="col u-hide--small">
+              {{ image(url="https://assets.ubuntu.com/v1/d68b3378-ubuntu-frame-final.png",
+                        alt="",
+                        width="852",
+                        height="172",
+                        hi_def=True,
+                        loading="lazy") | safe
+              }}
```
Comment (muhammad-ali-pk) [inline templates/internet-of-things/smart-displays.html:225]:
```suggestion
              {{ image(url="https://assets.ubuntu.com/v1/d68b3378-ubuntu-frame-final.png",
                            alt="",
                            width="852",
                            height="172",
                            hi_def=True,
                            loading="lazy") | safe
              }}
```

File: templates/internet-of-things/smart-displays.html [hunk 7]
```diff
@@ -7,377 +7,541 @@
...
+          <div class="row--25-75">
+            <hr class="p-rule--muted" />
+            <div class="col u-hide--small">
+              {{ image(url="https://assets.ubuntu.com/v1/9d527111-ubuntu-core-final.png",
+                        alt="",
+                        width="852",
+                        height="166",
+                        hi_def=True,
+                        loading="lazy") | safe
+              }}
```
Comment (muhammad-ali-pk) [inline templates/internet-of-things/smart-displays.html:254]:
```suggestion
              {{ image(url="https://assets.ubuntu.com/v1/9d527111-ubuntu-core-final.png",
                            alt="",
                            width="852",
                            height="166",
                            hi_def=True,
                            loading="lazy") | safe
              }}
```

**Review by muhammad-ali-pk — CHANGES_REQUESTED** [review]

**Comment by muhammad-ali-pk** [comment]
Looks great @britneywwc ! Just a few tiny corrections.

File: templates/internet-of-things/smart-displays.html (hunk 4 above)
Comment (britneywwc) [reply templates/internet-of-things/smart-displays.html]: Thanks for the suggestion! The tiered list pattern is using 50/50 split on medium and small screens, but this section is using 25/75 so it's slightly different. I will leave it as is for now

**Pull request merged**